viewer/vueapp/src/components/connections/Connections.vue

```diff
@@ -254,6 +236,61 @@
           </template>
         </b-dropdown> <!-- /link fields button -->
 
+        <!-- network baseline time range -->
```

Comment:
There are two new GUI controls which deal with this baseline feature:

- Baseline
  - disabled
  - 1 × actual time frame
  - 2 × actual time frame
  - 4 × actual time frame
  - 6 × actual time frame
  - 8 × actual time frame
  - 10 × actual time frame
  - 1 hour
  - 6 hours
  - 24 hours
  - 48 hours
  - 72 hours
  - 1 week
  - 2 weeks
  - 1 month
  - 2 months
  - 6 months
  - 1 year

- Baseline Visibility
  - All nodes
  - Actual nodes
  - Baseline nodes
  - New nodes only
  - Baseline nodes only

The first item of each of those dropdowns is selected by default, which results in the old behavior for the connections graph.

When **Baseline Time Frame**'s selection changes, the `cancelAndLoad` so that the query is re-run. Changing the **Baseline Node Visibility** doesn't re-run anything, as the existing data is enough (node/link visibility is just modified).